pkg/cmd/psa/api.go

```diff
@@ -17,12 +20,18 @@ type PodSecurityViolation struct {
 	// PodName is the name of the pod with the shortest name that violates the
 	// PodSecurity level.
 	PodName string `json:"podName"`
+
 	// Violations lists the violations that all the pods in the namespace made.
 	Violations []string `json:"violations"`
 	// Pod is the pod with the shortest name that violates the PodSecurity level.
 	Pod *corev1.Pod `json:"pod,omitempty"`
-	// PodController is the controller that manages the pod referenced.
-	PodControllers []any `json:"podcontroller,omitempty"`
+	// PodControllers is the controller that manages the pod referenced.
+	PodControllers []any `json:"podControllers,omitempty"`
+
+	// Labels contain the labels of interest, present in the namespace.
```

Comment:
what are "labels of interest" concretely? Only PSa labels? If so, please mention it.